docs/versions.md

```diff
@@ -38,21 +38,25 @@ most target .NET Standard 1.1.
 [1.6]: versions/netstandard1.6.md
 [2.0]: versions/netstandard2.0.md
 
-**NOTE**: The table above reflects the mappings that will happen when we release the
-.NET Standard 2.0 tooling. You can see that .NET Framework 4.6.1 mapping is being moved from
-1.4 to 2.0.
-
 ## How do I know which .NET Standard version I should target?
 
 When choosing a .NET Standard version, you should consider this trade-off:
 
 * The higher the version, the more APIs are available to you.
 * The lower the version, the more platforms implement it.
 
-So generally speaking, you should target the lowest version you get away with.
+So generally speaking, consider the following guidelines:
+
+* You should target the lowest version you get away with.
+* If you target .NET Standard 1.x, add .NET Standard 2.0 as [another target][multi-target].
```

Comment:
We should mention the concrete benefits of this:
- The dependency graph for .NET Standard 2.0 dependencies is greatly simplified.
- Consumers of the package running on .NET Standard 2.0 compatible frameworks will need to download less package dependencies as a result.

For example, here is JSON.NET:

![image](https://user-images.githubusercontent.com/95136/43678513-6a115926-97c9-11e8-9e4b-64cf06c61fcb.png)

Reply:
Update the document based on your feedback.

Reply:
Update the document based on your feedback.